src/content/docs/quick-installation-new-operator.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: docs
+title: Quick Installation(v2)
+permalink: /docs/getting-started/quick-installation-new-operator
+style_class: quick-installation
+---
+## disclaimer
+
+Version 2.0.0 represents an alpha release, exclusively accessible via the "fast" channel.
+Subsequent releases will transition to the "rolling" channel once the new operator attains greater stability.
```

Comment:
Correct me if I'm wrong but when the 2.x operator is stable enough to deploy all of the existing ODH components, then we would deprecate 'rolling` and make 'fast' the only channel.

Reply:
Yes, we'll update this to `fast`

Reply:
updated